tests/test_models.py

```diff
@@ -23,6 +23,13 @@ def __test_active_theme(self):
         self.assertTrue(theme.active)
         self.assertEqual(Theme.objects.filter(active=True).count(), 1)
 
+    def __test_active_dark_theme(self):
+        theme = Theme.objects.get_active(use_dark=True)
+        print(theme)
+        self.assertTrue(theme is not None)
```

Comment:
You get better error messages with specialized assert methods:
```suggestion
        self.assertIsNotNone(theme)
```